proto/decentraland/social_service_v2/social_service.proto

```diff
@@ -13,6 +13,11 @@ message InternalServerError {}
 
 message User { string address = 1; }
 
+message Pagination {
```

Comment:
We must consider that on requesting data that is paginated, the response should also include how many total items exists, so we can calculate how many pages we got